src/Microsoft.OpenApi.Hidi/Options/CommandOptions.cs

```diff
@@ -16,7 +16,7 @@ internal class CommandOptions
         public readonly Option<bool> CleanOutputOption = new("--clean-output", "Overwrite an existing file");
         public readonly Option<string> VersionOption = new("--version", "OpenAPI specification version");
         public readonly Option<string> MetadataVersionOption = new("--metadata-version", "Graph metadata version to use.");
-        public readonly Option<OpenApiFormat?> FormatOption = new("--format", "File format");
+        public readonly Option<string?> FormatOption = new("--format", "File format");
```

Comment:
```suggestion
        public readonly Option<string> FormatOption = new("--format", "File format");
```

(to align with the other string options)